packages/core/src/Stores/segment-store.js

```diff
@@ -0,0 +1,63 @@
+import {
+    action,
+    computed }                  from 'mobx';
+import BinarySocket             from '_common/base/socket_base';
+import { isLoginPages }         from '_common/base/login';
+import { get as getLanguage }   from '_common/language';
+import BaseStore                from './base-store';
+import { getAppId }             from '../config';
+
+export default class SegmentStore extends BaseStore {
+    // only available on production
+    is_applicable = /^(16929|19111)$/.test(getAppId());
+    language = getLanguage().toLowerCase();
```

Comment:
language could change after initialization of segmentstore, so better to use the function in `common_traits ` I think? (instead of having it as a property on the class)